doc/cephfs/cephfs-journal-tool.rst

```diff
@@ -23,9 +23,9 @@ Syntax
 
 ::
 
-    cephfs-journal-tool journal <inspect|import|export|reset>
-    cephfs-journal-tool header <get|set>
-    cephfs-journal-tool event <get|splice|apply> [filter] <list|json|summary|binary>
+    cephfs-journal-tool [options] journal <inspect|import|export|reset>
+    cephfs-journal-tool [options] header <get|set> <trimmed_pos|expire_pos|write_pos|pool_id> <value>
+    cephfs-journal-tool [options] event <get|splice|recover_dentries> [filter] <list|json|summary|binary>
```

Comment:
What are the possibilities for `[options]`?

Reply:
[These](https://github.com/ceph/ceph/pull/62860/files#diff-c9186667939f978da9959f3f4319a4af4d72d4f27dd704696e0e8a5add4546faR244) are those options.

Reply:
Ack.  Might I suggest a link to those here?

Reply:
It might be hard to accomplish. 
`::` is meant for literal blocks and it's not parsed except highlighting. So adding a `:ref:` would stay as a text in the literal block, not converting to a url.

Reply:
@anthonyeleven We could make it parsed-literal block without highlighting:
https://ceph--62860.org.readthedocs.build/en/62860/cephfs/cephfs-journal-tool/#syntax

Reply:
@anthonyeleven Which one would you prefer?

Reply:
@anthonyeleven I've made the url, as you've suggested. Please take a look.